src/py42/util.py

```diff
@@ -96,3 +98,29 @@ def to_list(value):
     if not isinstance(value, (list, tuple)):
         return [value]
     return value
+
+
+def parse_timestamp_to_milliseconds_precision(timestamp):
+    if isinstance(timestamp, int) or isinstance(timestamp, float):
+        return convert_timestamp_to_str(timestamp)
+
+    if isinstance(timestamp, datetime):
+        return convert_datetime_to_timestamp_str(timestamp)
+
+    if isinstance(timestamp, str):
+        return convert_datetime_to_timestamp_str(
+            datetime.strptime(timestamp, u"%Y-%m-%d %H:%M:%S")
+        )
+
+
+def parse_timestamp_to_microseconds_precision(timestamp):
```

Comment:
Copy some of the helper method calls from the method above to get this to work correctly.  See https://github.com/code42/py42/pull/205#issuecomment-698339164

Reply:
addressed.